driver-mapping/src/main/java/com/datastax/driver/mapping/MappingManager.java

```diff
@@ -213,17 +288,34 @@ public Session getSession() {
      * @return the accessor object for class {@code klass}.
      */
     public <T> T createAccessor(Class<T> klass) {
-        return getAccessor(klass);
+        return getAccessor(klass, defaultConfiguration);
+    }
+
+    /**
+     * Creates an accessor object based on the provided interface (that must be annotated by
+     * a {@link Accessor} annotation) with given {@code MapperConfiguration}.
+     * <p/>
+     * The {@code MappingManager} only ever keep one Accessor for each class, and so calling this
+     * method multiple time on the same class will always return the same object.
+     *
+     * @param <T>   the type of the accessor class.
+     * @param klass the (annotated) class for which to create an accessor object.
+     * @param configuration the configuration to be used.
+     * @return the accessor object for class {@code klass}.
+     */
+    public <T> T createAccessor(Class<T> klass, MapperConfiguration configuration) {
+        return getAccessor(klass, configuration);
     }
 
     @SuppressWarnings("unchecked")
-    private <T> Mapper<T> getMapper(Class<T> klass) {
+    private <T> Mapper<T> getMapper(Class<T> klass, MapperConfiguration configuration) {
+        configuration = new MapperConfiguration(configuration);
```

Comment:
I agree.
We can easily implement a key containing both class and config with `equals` method and make sure it won't happen.

Whichever we like better.